tools/resocsec

```diff
@@ -141,6 +145,13 @@ def resocsec():
     bl1.add_argument('--sign-stream', help="Helper for signing via stdio")
     bl1.add_argument('--sign-file', help="Helper for signing files")
 
+    spl_extra_sram_help='''AST2600 A1 and beyond has additional 24KB SRAM
+    where the SPL stack can run. If this is the case, this switch allows
+    the SPL to occupy the entire 64KB (signature included).'''
+    bl1.add_argument('--spl_extra_sram', dest='spl_extra_sram',
```

Comment:
See my comment in the pull-req about the option name, but also, please don't use underscores in option names for `resocsec` (you'll notice the underscores are inconsistent with the rest of the optional arguments in `resocsec`, and is something you reviewed me on with my original patch integrating the option into `socsec`).

The design point of `resocsec` was to align `socsec` with widely-accepted command-line UX conventions. As part of this effort `resocsec` uses `-` (dash) as a more conventional and ergonomic word separator in optional argument names.

Reply:
excellent point, will address this..